Framework/Kernel/test/ErrorReporterTest.h

```diff
@@ -51,9 +51,9 @@ class ErrorReporterTest : public CxxTest::TestSuite {
     reader.parse(message, root);
     auto members = root.getMemberNames();
     const std::vector<std::string> expectedMembers{
-        "ParaView", "application", "host",       "mantidSha1", "mantidVersion",
```

Comment:
As we're leaving the ParaView key in I think we should still check for it in the test?

Reply:
Should be back in now, great catch :+1: